rest/src/main/resources/scripts/ZStack SDK Go 开发规范与标准.md

```
@@ -0,0 +1,1096 @@
+        fmt.Println("VM not found")
+    } else {
+        fmt.Printf("Error: %v\n", err)
+    }
+}
+```
+
+---
+
+## 1. 项目结构
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)